docs/installation.rst

```diff
@@ -9,74 +9,68 @@ Requirements
 
 sunpy requires Python 3.8 or higher.
 
-Installing Scientific Python and sunpy
-======================================
+Installing Scientific Python and ``sunpy``
```

Comment:
I would argue that this is the wrong form of SunPy. This section of the docs is applicable to all coordinated packages as well as core.

Reply:
I can just drop the sunpy all together?

Reply:
and more than that, I think it *should* be the one place for all those packages. Perhaps we should extend the install section to account for that as well?

Reply:
You mean, for ndcube or sunkit-*?

Reply:
I mean in those packages we should be able to link here for setup? Maybe something along the lines of "You can conda install ndcube with `conda install -c conda-forge ndcube` see the sunpy installation instructions to get started"

So we should just make it clear where the breakpoint in these docs are for people linked here in that fashion?

Reply:
makes sense, I will see what I can butcher together.